src/AWS/Lambda/Combinators.hs

```diff
@@ -13,11 +13,12 @@ They map functions (instead of values) to turn basic handlers into handlers comp
 
 module AWS.Lambda.Combinators (
     withoutContext,
-    withInfallibleParse
+    withInfallibleParse,
+    withInfallibleParseEither
 ) where
 
-import           Data.Aeson             (FromJSON, parseJSON, Value)
-import           Data.Aeson.Types       (parseEither)
+import           Data.Aeson       (FromJSON, Value, parseJSON)
+import           Data.Aeson.Types (parseEither)
```

Comment:
Roll this change back to keep diff noise down.

Reply:
This is the result of running `stylish-haskell`, as suggested in `CONTRIBUTING.md`

Reply:
Apologies, this is my fault for never putting a build check in for formatting.  Without it, missing formatting kept slipping through.  The real intent behind the formatting was less about style and more about minimizing diff noise.  If you don't mind, that would be the preference at this point, and at some point I'll do a pass that cleans it all up and gets a formatting check in place to make sure it doesn't slip again.  I'll update the contributing doc shortly to reflect this preference, you adhering to it is much appreciated!

Reply:
Reverted all unnecessary import changes.